documentation/modules/proc-view-link-routes-qdstat.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// assemblies/assembly-qdstat.adoc
+
+[id='view-link-routes-qdstat-{context}']
+= Viewing link routes using qdstat
+
+Two link routes are created by default to manage last will and testament. You can view these link routes using `qdstat`.
```

Comment:
These link routes only apply when MQTT is enabled so they are not always created by default. So again, I'd say just leave out the explanation of the default link routes.

> You can view the link routes using `qdstat`.